config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml

```diff
@@ -325,105 +325,71 @@ periodics:
   labels:
     preset-service-account: "true"
     preset-k8s-ssh: "true"
-    preset-bazel-scratch-dir: "true"
-    preset-pull-kubernetes-e2e: "true"
-    preset-pull-kubernetes-e2e-gce: "true"
   spec:
     containers:
     - args:
-      - --root=/go/src
-      - --repo=k8s.io/kubernetes=$(PULL_REFS)
```

Comment:
We don't need to specify the repo?

Reply:
At least in from https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L316, no repo is set.